user/models.py

```diff
@@ -0,0 +1,11 @@
+from django.db import models
+
+class User(models.Model):
+    first_name      = models.CharField(max_length = 64, null  = False)
+    last_name       = models.CharField(max_length = 64, null  = False)
+    email           = models.EmailField(unique    = True)
+    phone           = models.CharField(max_length = 64, null  = False, unique = True)
+    password        = models.CharField(max_length = 128, null = False)
+    birth           = models.CharField(max_length = 128, null = True)
+    newsletter      = models.BooleanField()
+    top_contributor = models.BooleanField()
```

Comment:
```suggestion
    first_name      = models.CharField(max_length = 64, null = False)
    last_name       = models.CharField(max_length = 64, null = False)
    email           = models.EmailField(unique = True)
    phone           = models.CharField(max_length = 64, null = False, unique = True)
    password        = models.CharField(max_length = 128, null = False)
    birth           = models.CharField(max_length = 128, null = True)
    newsletter      = models.BooleanField()
    top_contributor = models.BooleanField()
```
property 까지 라인 맞추시지 않으셔도 됩니다 😅

Reply:
전체 블록 지정해서 Align 먹이다보니;;;
일단 수정했습니다.